src/rules/consistent-list-newline.ts

```diff
@@ -32,7 +32,28 @@ export default createEslintRule<Options, MessageIds>({
       recommended: 'stylistic',
     },
     fixable: 'whitespace',
-    schema: [],
+    schema: [{
+      type: 'object',
+      properties: {
+        ArrayExpression: { type: 'boolean' },
+        ArrowFunctionExpression: { type: 'boolean' },
+        CallExpression: { type: 'boolean' },
+        ExportNamedDeclaration: { type: 'boolean' },
+        FunctionDeclaration: { type: 'boolean' },
+        FunctionExpression: { type: 'boolean' },
+        ImportDeclaration: { type: 'boolean' },
+        NewExpression: { type: 'boolean' },
+        ObjectExpression: { type: 'boolean' },
+        TSInterfaceDeclaration: { type: 'boolean' },
+        TSTupleType: { type: 'boolean' },
+        TSTypeLiteral: { type: 'boolean' },
+        TSTypeParameterDeclaration: { type: 'boolean' },
+        TSTypeParameterInstantiation: { type: 'boolean' },
+        ObjectPattern: { type: 'boolean' },
+        ArrayPattern: { type: 'boolean' },
+      },
```

Comment:
```suggestion
      } satisfies Record<keyof Options[0], any>,
```

Can you add a constrain here so we don't accidentally miss an option when we change the list later?

Reply:
done